integration/testrunner/db.go

```diff
@@ -42,11 +45,31 @@ func queryAuditLog(ctx context.Context, tb testing.TB, query *bigquery.Query) ([
 	if err != nil {
 		return nil, fmt.Errorf("failed to read job: %w", err)
 	}
-	var row []bigquery.Value
-	if err = it.Next(&row); err != nil {
-		return nil, fmt.Errorf("failed to get next row: %w", err)
+	var results []logpb.LogEntry
```

Comment:
```suggestion
	var results []*logpb.LogEntry
```

Always prefer pointers